.cirrus.yml

```diff
@@ -400,3 +400,13 @@ asciidoctor_warnings_task:
         ASCIIDOCTOR_OPTS: '--failure-level WARN'
 
     script: make doc
+
+i18nspector_task:
+    name: .po files
+
+    container:
+      cpu: 1
+      memory: 512MB
+      dockerfile: docker/i18nspector.dockerfile
+
+    script: make run-i18nspector | tee i18nspector.log && ! egrep --silent "^(E|W):" i18nspector.log
```

Comment:
Just out of curiosity, what does the exclamation mark do? (in `! egrep …`)

Reply:
It negates the exit code of the command. `grep` returns 0, i.e. "false", when it found something, but I want exit code 1 instead.

Perhaps `egrep ... && exit 1` would be more obvious? Or should I just write a comment? I admit this oneliner is a bit complicated, I should probably document it better.

Reply:
I went ahead and did both: 2271bbc uses `&& exit 1` *and* adds comments to explain what's going on.

Reply:
Ah, of course, `&& exit 1` doesn't work because the build will *always* exit with 1 now: either `egrep` will fail, or the `exit 1` will. In 8dd8825, I moved back to the exclamation mark operator.

Reply:
Looks good, thanks!